source/js/donate-modal/dnt.js

```diff
@@ -0,0 +1,18 @@
+let _dntStatus = navigator.doNotTrack || navigator.msDoNotTrack,
```

Comment:
should we just move the existing `/source/js/buyers-guide/dnt.js` to `/source/js/dnt.js` so every module and class on the site can utilize it?

Reply:
yes!